routes/api.php

```diff
@@ -25,3 +27,27 @@
 Route::fallback(function () {
     return redirect()->to('https://opentransportmanager.github.io/otm-docs/');
 });
+
+Route::post('/login', function (Request $request) {
```

Comment:
Oh, this is wrong on so many levels. Please:

- move everything to controller
- move validation to form request
- throw an exception and handle it in Handler
- use `response()->json($data, 201)` 
- maybe this array with user and token could be replaced by some object?
- replace `201` with proper Response-class const.